src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -83,11 +83,16 @@ const ColonyHome = ({ match, location }: Props) => {
   const { walletAddress } = useLoggedInUser();
   const { version: networkVersion } = useNetworkContracts();
 
-  const { domainFilter } = parseQS(location.search) as {
+  const { domainFilter: queryDomainFilterId } = parseQS(location.search) as {
     domainFilter: string | undefined;
   };
-  const filteredDomainId = domainFilter
-    ? parseInt(domainFilter, 10) || COLONY_TOTAL_BALANCE_DOMAIN_ID
+
+  const [domainIdFilter, setDomainIdFilter] = useState<number>(
+    Number(queryDomainFilterId),
+  );
+
+  const filteredDomainId = domainIdFilter
+    ? domainIdFilter || COLONY_TOTAL_BALANCE_DOMAIN_ID
```

Comment:
```suggestion
   const filteredDomainId = domainIdFilter || COLONY_TOTAL_BALANCE_DOMAIN_ID
```

Reply:
Nice catch. 

This had other logic in it initially, but apparently, I failed to clean it up properly.